src/Sulu/Bundle/ContentBundle/Resources/js/views/WebspaceOverview/WebspaceOverview.js

```diff
@@ -82,7 +82,9 @@ class WebspaceOverview extends React.Component<ViewProps> {
         router.bind('webspace', this.webspace);
         apiOptions.webspace = this.webspace;
 
-        this.datagridStore = new DatagridStore('nodes', observableOptions, apiOptions);
+        apiOptions.fields = 'title,published';
```

Comment:
What exactly do you need this for?

Reply:
Without this `fields` parameter the API isn't working...